### DISCUSSION TITLE
正则编辑器的'匹配字符串'和'匹配结果'建议左右排列布局,这样更加直观类似json路径的布局

### DISCUSSION THREAD

**Discussion by gauson89** [he3-app/he3-feedback#117] [Ideas]
正则编辑器的'匹配字符串'和'匹配结果'建议左右排列布局,这样更加直观类似json路径的布局

**Comment by lyzhang1999** [comment]
感谢您的建议，我已建立 issues 来跟踪这个问题：https://github.com/he3-app/he3-feedback/issues/120
